Game/Game.gd

```diff
@@ -12,6 +12,11 @@ func _ready():
 		create_player(get_tree().get_network_unique_id())
 
 
+func _process(_delta):
+	if Input.is_action_pressed("exit"):
+		get_tree().quit()
```

Comment:
This should be checked for in `_unhandled_key_input`, which is faster and allows escape to still be used in UI.